src/distro/distro.py

```diff
@@ -55,7 +55,7 @@
     # Python 3.7
     TypedDict = dict
 
-__version__ = "1.8.0"
+__version__ = "1.9.0"
```

Comment:
I've got a notification about a `v1.9.0` tag being created, but no release was pushed on PyPI : have you planned to delete/recreate this tag when this PR will be merged to `master` for consistency ?

Reply:
Yeah, that was my bad. I created the tag and pushed it to test some deployment mechanism I was testing, but pushed it and forgot to delete it. I'll delete the tag now, fix, merge, then push the tag.